__tests__/apiPostArticleComment.test.js

```diff
@@ -0,0 +1,79 @@
+const request = require("supertest");
+const app = require("../index");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+const jestSorted = require("jest-sorted");
+
+afterAll(() => {
+  db.end();
+});
+
+beforeEach(() => {
+  return seed(data);
+});
+
+describe("POST /api/articles/:article_id/comments", () => {
+  it("checking the body of returned comment", async () => {
+    const commentBody = "small cat";
+    const username = "lurker";
+    const articleId = 4;
+
+    const response = await request(app)
+      .post(`/api/articles/${articleId}/comments`)
+      .send({ username: username, body: commentBody });
+
+    expect(response.status).toEqual(201);
+    expect(response.body).toBeInstanceOf(Object);
+
+    expect(response.body).toEqual({ comment: "small cat" });
+  });
+});
+
+it("returns 404 if the article_id is not a number", async () => {
+  const commentBody = "small cat";
+  const username = "lurker";
+  const response = await request(app)
+    .post("/api/articles/notANumber/comment")
+    .send({ username, body: commentBody })
+    .catch((err) => {
+      expect(err.response.status).toBe(400);
+    });
```

Comment:
We should be checking the response thats come back, not the catch. 
Even with error messages and status, it should still be a response. 
You need to check the status of the response as well as the message you've sent back - same for all these ones